etc/rucio.json

```diff
@@ -0,0 +1,15 @@
+{
+    "name": "lta.transfer.rucio.RucioTransferService",
+    "account": "root",
+    "password": "hunter2",
```

Comment:
I have an objection to putting even fake passwords into git.

Reply:
Move this to an `example` directory and replace the specifics with examples.